docs/openapi/resources/credential-issuer.yml

```diff
@@ -13,6 +13,9 @@ post:
       application/json:
         schema:
           type: object
+          required:
+            - credential
+            - options
```

Comment:
isn't this supposed to be optional?

Reply:
hmm. I may be misunderstanding this. What would the expected behavior be if a credential issue request is made without credentials to sign, or without specifying the signature type?